src/scripts.js

```diff
@@ -236,10 +245,10 @@ function addWalkingStats(user, userStorage, date, activityRepo) {
   const stepGoalCard = document.getElementById('stepGoalCard');
   const milesWalked = document.getElementById('milesWalked');
   const avgStepGoalCard = document.getElementById('avgStepGoalCard');
-  userStrideLength.innerText = `${user.strideLength}meters`;
+  userStrideLength.innerText = `${user.strideLength}m`;
   stepGoalCard.innerText = `${user.dailyStepGoal}`;
   avgStepGoalCard.innerText = `${userStorage.calculateAverageStepGoal()}`;
-  milesWalked.innerText = `${activityRepo.getMilesFromStepsByDate(user.id, date, user)}`;
+  milesWalked.innerText = `${activityRepo.getMilesFromStepsByDate(user.id, date, user)}mi`;
 }
 
 
```

Comment:
LOVE LOVE LOVE the thick black text instead of the circles. 
We should add more white space to the bottom two graph sections - top and bottom margins for all graphs.